pkgs/development/python-modules/gremlinpython/default.nix

```diff
@@ -33,13 +35,14 @@ buildPythonPackage rec {
     sed -i '/pytest-runner/d' setup.py
 
     substituteInPlace setup.py \
-      --replace 'importlib-metadata<5.0.0' 'importlib-metadata' \
-      --replace "os.getenv('VERSION', '?').replace('-SNAPSHOT', '.dev-%d' % timestamp)" '"${version}"'
+      --replace-fail 'importlib-metadata<5.0.0' 'importlib-metadata' \
```

Comment:
This could benefit from the `pythonRelaxDepsHook`.

Reply:
Tried, doesn't seem to do the job.